docs/changelog.rst

```diff
@@ -4,6 +4,10 @@
 Changelog
 =========
 
+- :release:`10.5.2 <4th January 2024>`
```

Comment:
This is a new feature with no breaking changes, so lets bump the minor version, rather than patch